src/style.scss

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2022 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
 // See LICENSE in the project root for license information.
 
-/* stylelint-disable max-nesting-depth */
+/* stylelint-disable max-nesting-depth, declaration-no-important */
```

Comment:
Just noticed this shouldn't have been moved to the whole file. We don't want to normally allow this and where it was before scoped the exclusion to only the specific code we cared to exclude.